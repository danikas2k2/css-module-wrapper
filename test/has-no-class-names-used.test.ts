import path from 'path';
import { compile } from './utils/compile';
import { DIR } from './utils/ufs';

it('has no class names used', async () => {
    const [output] = await compile({
        entry: path.resolve(DIR, 'has-no-class-names-used.tsx'),
    });
    expect(output).toMatchSnapshot();
});
