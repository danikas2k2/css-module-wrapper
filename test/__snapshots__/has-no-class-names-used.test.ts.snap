// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`has no class names used 1`] = `
""use strict";
import { jsx } from "react/jsx-runtime";
import classNames__bind from "classnames/bind.js";
import ___Button_css_ from "./Button.css";
const classNames = classNames__bind.bind(___Button_css_);
export function Button({
  color = "neutral",
  variant = "solid",
  size = "small",
  spacing = "default",
  disabled,
  ...props
}) {
  return /* @__PURE__ */ jsx("button", { disabled, "aria-disabled": disabled, ...props });
}
"
`;
